src/ucp/wireup/wireup_cm.c

```diff
@@ -17,15 +17,19 @@
 #include <ucs/sys/string.h>
 
 
-/* if CM callback is invoked for CM lane of UCP EP which is marked as
- * failed one, i.e. the failing UCP is in progress, because UCT CM EP is
- * still alive (e.g. err_handling progress part was scheduled on worker
- * or discarding) - so, don't do any actions in the callback -
- * discarding functionality will destroy UCT EP */
-#define UCP_CM_CB_CHECK_UCP_EP(_ucp_ep, _uct_cm_ep, _exit_action) \
+/**
+ * @brief Check whether CN callback should be called or not.
+ *
+ * @param [in]    _ucp_ep        UCP Endpoint for which CM callback is called.
+ * @param [in]    _uct_cm_ep     UCT CM Endpoint which calls CM callback.
+ * @param [in]    _failed_action Action to do if UCP EP is in a FAILED state.
```

Comment:
minor: can remove some spaces after [in]

Reply:
fixed